code/modules/awaymissions/mission_code/snowdin.dm

```diff
@@ -0,0 +1,281 @@
+				/obj/item/weapon/c4 = 7,
+				/obj/item/weapon/grenade/clusterbuster/smoke = 15,
+				/obj/item/clothing/under/chameleon = 13,
+				/obj/item/clothing/shoes/sneakers/syndigaloshes = 10,
+				/obj/item/borg/upgrade/ddrill = 3,
+				/obj/item/borg/upgrade/soh = 3)
+
+/obj/effect/spawner/lootdrop/snowdin/dungeonmid
+	name = "dungeon mid"
+	loot = list(/obj/item/weapon/defibrillator/compact = 6,
+				/obj/item/weapon/storage/firstaid/tactical = 35,
+				/obj/item/weapon/teleportation_scroll/apprentice = 4,
+				/obj/item/weapon/shield/energy = 6,
+				/obj/item/weapon/shield/riot/tele = 12,
+				/obj/item/weapon/dnainjector/lasereyesmut = 7,
+				/obj/item/weapon/gun/magic/wand/fireball/innert = 3,
+				/obj/item/weapon/pneumatic_cannon = 15,
+				/obj/item/weapon/melee/energy/sword = 7,
+				/obj/item/weapon/spellbook/oneuse/knock = 15,
+				/obj/item/weapon/spellbook/oneuse/summonitem = 20,
+				/obj/item/weapon/spellbook/oneuse/forcewall = 17,
+				/obj/item/weapon/storage/backpack/holding = 12,
+				/obj/item/weapon/grenade/spawnergrenade/manhacks = 6,
+				/obj/item/weapon/grenade/spawnergrenade/spesscarp = 7,
+				/obj/item/weapon/grenade/clusterbuster/inferno = 3,
+				/obj/item/stack/sheet/mineral/diamond{amount = 15} = 10,
+				/obj/item/stack/sheet/mineral/uranium{amount = 15} = 10,
+				/obj/item/stack/sheet/mineral/plasma{amount = 15} = 10,
+				/obj/item/stack/sheet/mineral/gold{amount = 15} = 10,
+				/obj/item/weapon/spellbook/oneuse/barnyard = 4,
+				/obj/item/weapon/pickaxe/drill/diamonddrill = 6,
+				/obj/item/borg/upgrade/vtec = 7,
+				/obj/item/borg/upgrade/disablercooler = 7)
+
+
+/obj/effect/spawner/lootdrop/snowdin/dungeonheavy
+	name = "dungeon heavy"
+	loot = list(/obj/item/weapon/twohanded/singularityhammer = 25,
+				/obj/item/weapon/twohanded/mjollnir = 10,
+				/obj/item/weapon/twohanded/fireaxe = 25,
+				/obj/item/organ/internal/brain/alien = 17,
+				/obj/item/weapon/twohanded/dualsaber = 15,
+				/obj/item/organ/internal/heart/demon = 7,
+				/obj/item/weapon/gun/projectile/automatic/c20r/unrestricted = 16,
+				/obj/item/weapon/teleportation_scroll = 5,
+				/obj/item/weapon/dice/d20/fate = 4,
+				/obj/item/weapon/gun/magic/wand/resurrection/innert = 15,
+				/obj/item/weapon/gun/magic/wand/resurrection = 10,
+				/obj/item/device/uplink/old = 2,
+				/obj/item/weapon/spellbook/oneuse/charge = 12,
+				/obj/item/weapon/grenade/clusterbuster/spawner_manhacks = 15,
+				/obj/item/weapon/spellbook/oneuse/fireball = 10,
+				/obj/item/weapon/pickaxe/drill/jackhammer = 30,
+				/obj/item/borg/upgrade/syndicate = 13,
+				/obj/item/borg/upgrade/selfrepair = 17)
+
+/obj/effect/spawner/lootdrop/snowdin/dungeonmisc
+	name = "dungeon misc"
+	lootdoubles = 2
+	lootcount = 1
+
+	loot = list(/obj/item/stack/sheet/mineral/snow{amount = 25} = 10,
+				/obj/item/toy/snowball = 15,
+				/obj/item/weapon/shovel = 10,
+				/obj/item/weapon/twohanded/spear = 8,
+				)
+
+//special items//--
+
+/obj/item/clothing/belt/
```

Comment:
i think that was a loot item i was thinking of making but didn't bother finishing, guess i forgot to remove the path